Services/DataBaseService/BrokerConsumers/DeleteUserConsumer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MassTransit;
+using DTO.BrokerRequests;
+using System.Threading.Tasks;
+using DataBaseService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using DTO;
+
+namespace DataBaseService.BrokerConsumers
+{
+    public class DeleteUserConsumer : IConsumer<InternalDeleteUserRequest>
+    {
+        private readonly IUserRepository userRepository;
+
+        public DeleteUserConsumer([FromServices] IUserRepository userRepository)
+        {
+            this.userRepository = userRepository;
+        }
+
+        private OperationResult DeleteUser(InternalDeleteUserRequest request)
+        {
+            userRepository.DeleteUser(request.Id);
```

Comment:
What will happen if  this user is already deleted. You are always return true as I understood

Reply:
I think it will return exception
 if(user.IsActive==false)
            {
                throw new ForbiddenException("User was deleted early");
            }

Reply:
I think an exception will be thrown. https://github.com/lanit-students/TradingStation/pull/60/files#diff-4b8e9f1d20a55719408ebc00a9583288R50